src/cswinrt/main.cpp

```diff
@@ -119,6 +119,7 @@ Where <spec> is one or more of:
         int result{};
         writer w;
 
+        /* Special case the usage exceptions to print CLI options */
```

Comment:
Is this comment correlated to a change?

Reply:
No I was reading the cswinrt main and felt like this comment would be useful. Can remove if you want

Reply:
You can leave it, was just initially confused as I didn't see a correlated change.